zmsslim/src/Slim/LanguageTranslator.php

```
@@ -25,7 +25,8 @@ public function __construct($fallbackLocale, $defaultLocale, $defaultLang)
 
         $this->defaultLang = $defaultLang;
 
-        $translatorClass = \APP::TRANSLATOR_CLASS;
+        /** @var class-string<Translator> $translatorClass */
+        $translatorClass = \App::TRANSLATOR_CLASS;
```

PHPStan (2.2.7)

[error] PHPDoc tag `@var` with type class-string<Symfony\Component\Translation\Translator> is not subtype of native type '\\Symfony\\Component…'.


(varTag.nativeType)